<launch>
   <node name="optimal_planner"
        pkg="heightmap_costmap"
        type="a_star.py"
        output="screen">

    <param name="total_cost_csv"
           value="$(find heightmap_costmap)/total_cost.csv" />
    <param name="resolution" value="0.05"/>
    <param name="origin_x" value="0.0"/>
    <param name="origin_y" value="0.0"/>


    <!-- Start and via points (pixel coordinates) -->
    <param name="start_x" value="442" />
    <param name="start_y" value="461" />
    <param name="via1_x" value="371" />
    <param name="via1_y" value="184" />
    <param name="via2_x" value="205" />
    <param name="via2_y" value="452" />
    <param name="via3_x" value="315" />
    <param name="via3_y" value="381" />
    <param name="via4_x" value="463" />
    <param name="via4_y" value="350" />
    <param name="via5_x" value="573" />
    <param name="via5_y" value="286" />
    <param name="via6_x" value="269" />
    <param name="via6_y" value="246" />
    <param name="via7_x" value="471" />
    <param name="via7_y" value="246" />
    <param name="via8_x" value="558" />
    <param name="via8_y" value="404" />
    <param name="via9_x" value="428" />
    <param name="via9_y" value="303" />


    <param name="output_csv" value="$(find heightmap_costmap)/path.csv"/>
   </node>
</launch>
